packages/web/src/components/gcds-card/gcds-card.css

```diff
@@ -18,50 +18,56 @@
 
 @layer default {
   :host .gcds-card {
-    border: var(--gcds-card-border);
-    border-radius: var(--gcds-card-border-radius);
     background-color: var(--gcds-card-background-color);
     display: block;
     padding: var(--gcds-card-padding);
     height: 100%;
     color: var(--gcds-card-color);
     position: relative;
+    box-shadow: var(--gcds-card-box-shadow);
+    overflow: hidden;
+    max-width: var(--gcds-card-max-width);
 
     > *:not(slot):not(.gcds-card__spacer) {
       display: block;
       margin: var(--gcds-card-margin);
     }
 
+    .gcds-badge {
+      background-color: var(--gcds-card-badge-background-color);
+      position: absolute;
+      top: 0;
+      left: 0;
+      padding: var(--gcds-card-badge-padding);
+      text-wrap: nowrap;
```

Comment:
Not sure how much you talked to design about this - but can we maybe find a solution for the potential text-overflow issue if the user enters long badge text? It looks a bit broken in the current overflow example. We could add an ellipsis if we want to keep it on one line or let it overflow to multiple lines. Or third option: set a character limit to ensure it fits on one line.

Current overflow:
![Screenshot 2024-07-15 at 4 27 54 PM](https://github.com/user-attachments/assets/43539237-99bb-4c03-b9c8-6de177944cd7)

Overflow onto multiple lines (for example):
![Screenshot 2024-07-15 at 4 31 07 PM](https://github.com/user-attachments/assets/25df6bd2-78b9-42db-9bab-36ba53579db3)

Reply:
Thanks for flagging this. I will bring it to dev/design sync

Reply:
Thank you for flagging. The intent of the badge on the card is to provide short and concise information to the user typically in the form of a single word or two words (short phrase). Common examples: "New", "Updated", "In stock", "Update available". Given this context the option of a character count would seem appropriate. (e.g. max 20 characters)

Reply:
Maybe rather than an arbitrary number like 20 characters, we can instead have the limit be closer to what a full line of text across would be?

Reply:
Validation for values over 20 characters has been added to the `badge` property